src/components/SlateEditor/plugins/audio/types.ts

```diff
@@ -13,6 +13,6 @@ export const TYPE_AUDIO = 'audio';
 
 export interface AudioElement {
   type: 'audio';
-  data?: AudioEmbedData;
+  data: AudioEmbedData;
```

Comment:
Håper denne oppdateringen er ok? Så at andre embed elements var definert sånn

Reply:
Den kan være litt skummel. Ser Jonas har skrevet en sjekk i `SlateAudio` i tilfelle den er undefined...

Reply:
Det må nok være ?. For den er potensielt undefined i vanlige artikler når et audio embed lages 🦀